src/components/Summary/AlterationSummary.vue

```diff
@@ -267,6 +290,11 @@ export default class AlterationSummary extends Mixins(
     await this.scrollToTop(document.getElementById('app'))
   }
 
+  // sum of alteration fees
+  get alterationFees (): number {
+    return this.getCurrentFees.filingFees + this.getCurrentFees.futureEffectiveFees
```

Comment:
Not a bad idea to protect yourself here if `getCurrentFees` is null or undefined. 
ie optional chain `this.getCurrentFees?` 
On that note, if this also is feeding a value to the UI for display, a fallback value might be required.

Reply:
Yeah, you're right. I changed it to check truthy for `this.getCurrentFees?.filingFees`. Otherwise it doesn't make sense to display anything for alterationFees.